books/models.py

```diff
@@ -26,6 +26,19 @@ class Book(models.Model):
     thumb = models.ImageField(upload_to="covers/", blank=True)
     published_date = models.DateTimeField(blank=True, null=True)
     last_updated = models.DateTimeField(auto_now_add=True, null=True)
+    isbn = models.CharField(
+        "ISBN",
+        max_length=13,
+        help_text='13 Character \
+        <a href="https://www.isbn-international.org/content/what-isbn">ISBN number</a>',
```

Comment:
This will render fine in HTML because it ignores extra space, but if you want to get your spacing right within a long string you can also do:
```python
help_text=(
    "13 Character "
    "<a href="https://www.isbn-international.org/content/what-isbn">"
    "ISBN number</a>"
)
```

the brackets here are not a tuple, they will just tell python to interpret it as "one".